release/test_bom.sh

```diff
@@ -123,6 +124,28 @@ function test_bom_copy_tld {
 		"$(ls test-dependencies/expected/META-INF)"
 }
 
+function test_bom_manage_bom_jar {
+	mkdir api-jar
+	mkdir --parents temp_dir_manage_bom_jar/META-INF
+
+	touch temp_dir_manage_bom_jar/META-INF/ECLIPSE_.RSA
+	touch temp_dir_manage_bom_jar/META-INF/ECLIPSE_.SF
+	touch temp_dir_manage_bom_jar/META-INF/test.txt
+
+	manage_bom_jar javax.persistence-2.2.0.jar &> /dev/null
+
+	mkdir --parents test-dependencies/expected/api-jar/META-INF
+
+	touch test-dependencies/expected/api-jar/META-INF/test.txt
+
+	assert_equals \
+		"$(ls api-jar/META-INF)" \
+		"$(ls test-dependencies/expected/api-jar/META-INF)"
+
+	rm --force --recursive api-jar
+	rm --force --recursive test-dependencies/expected/api-jar
+}
+
```

Comment:
@kiwm, this function isn't placed according the alphabetical order, please sort it.

In addition, please refactor this function based on https://github.com/brianchandotcom/liferay-docker/pull/938/commits/3f6689cdfb98c7f039ab2d78a07886978b2b350d.

Thanks